### DISCUSSION TITLE
Update token whitelist

### DISCUSSION THREAD

**Discussion by wu-clan** [fastapi-practices/fastapi_best_architecture#217] [Ideas]
Hi, @downdawn 

I want to add token blacklist function

Also, it seems more logical to change the whitelist function to token exclude

**Comment by downdawn** [comment]
What are the scenarios for token blacklist?

**Reply by wu-clan** [reply]
When creating a new token, the current token should expire immediately, with the option to blacklist or delete it directly

**Comment by wu-clan** [comment]
`TOKEN_WHITELIST` -> `TOKEN_EXCLUDE`

TOKEN_WHITELIST is currently only available for JWT authentication, for RBAC is not available, are we going to modify it?

**Reply by downdawn** [reply]
Both options are fine, but it's easier to communicate with a whitelist when working on the frontend.
login、captcha...

**Comment by wu-clan** [comment]
OK, I'll create a PR.

**Reply by wu-clan** [reply]
Merged: #220
